sphinx/util/osutil.py

```diff
@@ -219,7 +221,11 @@ def abspath(pathdir):
     # type: (unicode) -> unicode
     pathdir = path.abspath(pathdir)
     if isinstance(pathdir, bytes):
-        pathdir = pathdir.decode(fs_encoding)
+        try:
+            pathdir = pathdir.decode(fs_encoding)
+        except UnicodeError:
+            raise ConfigError('multibyte filename not supported on this '
```

Comment:
I must say it is very strange that `abspath()` raises `ConfigError`.

Reply:
@tk0miya I used this because it seemed like a configuration issue. However, it is not a configuration *file* issue. What would you suggest instead? Remember that we'll raise this exception for any and all issues with provided paths.

Reply:
No, this is not related with case of `Sphinx` class. this `abspath()` is an utility function. So it is used for other paths. So I think using standard exception is better. So I think it is better to raise `UnicodeError` directly.

Reply:
Done